src/extension/background-script/actions/lnurl/authOrPrompt.ts

```diff
@@ -33,12 +34,25 @@ async function authOrPrompt(
   // we have the check the unlock status manually. The account can still be locked
   // If it is locked we must show a prompt to unlock
   const isUnlocked = await state.getState().isUnlocked();
+  const account = await state.getState().getAccount();
+  const isAlbyOAuthConnector = isAlbyOAuthAccount(account?.connector);
 
   // check if there is a publisher and lnurlAuth is enabled,
   // otherwise we we prompt the user
-  if (isUnlocked && allowance && allowance.enabled && allowance.lnurlAuth) {
+
+  if (
+    isUnlocked &&
+    allowance &&
+    allowance.enabled &&
+    allowance.lnurlAuth &&
+    (!isAlbyOAuthConnector || account?.mnemonic)
+  ) {
     return await authFunction({ lnurlDetails, origin: message.origin });
   } else {
+    return await authPrompt();
+  }
+
+  async function authPrompt() {
```

Comment:
It is strange this method is declared at the very bottom after the return statements.

@pavanjoshi914 can you move it above?

Reply:
Edit: I will move it

Reply:
thanks